tests/Tests/Aggregations/Pipeline/SumBucket/SumBucketAggregationUsageTests.cs

```diff
@@ -46,7 +46,7 @@ public class SumBucketAggregationUsageTests : AggregationUsageTestBase<ReadOnlyC
 				date_histogram = new
 				{
 					field = "startedOn",
-					interval = "month",
+					calendar_interval = "month",
```

Comment:
Why was this changed in the structure?

Reply:
interval was marked as obsolete and was therefore removed. Reference: https://github.com/elastic/elasticsearch-net/blob/0b2a83b8f9647ae482c91e67fe9bf983d18c2947/src/Nest/Aggregations/Bucket/DateHistogram/DateHistogramAggregation.cs#L45